Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -11,12 +11,15 @@ import UserNotifications
 import XCTest
 
 final class AppDelegateViewModelTests: TestCase {
-  let vm: AppDelegateViewModelType = AppDelegateViewModel()
+  var vm: AppDelegateViewModelType!
```

Comment:
Can we make this `fileprivate` too? And why the need to re-initialize the VM on `setup`?

Reply:
Needed to do this in order to ensure that the scheduler is replaced in the environment before the test runs. We've had to do this in a few VMs. We can probably undo this depending how we change the survey display logic.